float32 torque1
float32 torque2
float32 torque3
float32 force1
float32 force2
float32 force3